L1Trigger/L1TTwinMux/plugins/L1TTwinMuxParamsESProducer.cc

```diff
@@ -0,0 +1,104 @@
+// -*- C++ -*-
+//
+// Class:      L1TTwinMuxParamsESProducer
+//
+// Original Author:  Giannis Flouris
+//         Created:
+//
+//modifications: g karathanasis
+
+
+// system include files
+#include <memory>
+#include "boost/shared_ptr.hpp"
+
+// user include files
+#include "FWCore/Framework/interface/ModuleFactory.h"
```

Comment:
Please check if all of these includes are needed.

Reply:
Hi Thomas,

I checked and removed some. The specific one is needed